events/event_generators.py

```diff
@@ -475,8 +475,11 @@ def generateExitMetastakeEvent(context: Context, user: Account, metastake_contra
 
         decoded_metastake_tk_attributes = get_lp_from_metastake_token_attributes(metastake_token_attributes)
 
-        farm_tk_details = context.network_provider.api.get_non_fungible_token(
-            metastake_contract.farm_token, decoded_metastake_tk_attributes['lp_farm_token_nonce']
+        # farm_tk_details = context.network_provider.api.get_non_fungible_token(
```

Comment:
maybe remove commented code

Reply:
I wasn't sure whether I will have to get back to this and still am a bit. Proxy requires the address of an user holding the tokens to just read the token attributes so I had to do some changes to accommodate this and I don't really like it. Ideally I should've been able to use it as in API without necessarily passing an user, but will have to see.

Reply:
I will probably add a todo comment with this note, to know why I did this.